src/angularjs/src/styles/components/_card.scss

```diff
@@ -19,6 +19,30 @@
   }
 }
 
+.pfb-stripe {
```

Comment:
Can this class be re-used for the cards in the "how is this score calculated" section below the map?